components/cliniko/cliniko.app.js

```diff
@@ -0,0 +1,63 @@
+const { axios } = require("@pipedreamhq/platform");
+
+module.exports = {
+  type: "app",
+  app: "cliniko",
+  propDefinitions: {
+    patientId: {
+      type: "integer",
+      label: "Patient ID",
+      description: "Enter a unique patient ID.",
+    },
+  },
+  methods: {
+    /**
+     * Get the Cliniko API key for the authenticated user.
+     * @returns {String} The Cliniko API key.
+     */
+    _apiKey() {
+      return this.$auth.api_key;
+    },
+    _shard() {
+      return this.$auth.shard;
+    },
+    _baseApiUrl() {
+      return `https://api.${this._shard()}.cliniko.com/v1`;
+    },
+    _makeRequestConfig(url) {
+      const auth = {
+        username: this._apiKey(),
+        password: "", // No password needed.
+      };
+      // Cliniko requires contact email address in User-Agent header.
+      // See https://github.com/redguava/cliniko-api#identifying-your-application.
+      const headers = {
+        "Accept": "application/json",
+        "User-Agent": "Pipedream (cliniko@pipedream.com)",
```

Comment:
Ignore my previous comment, I didn't see your note re: the contact email address.

Can you set the email to support@pipedream.com?

Reply:
Ha. Yes, I originally used `support@pipedream`, then I switched it because I saw that `cliniko@pipedream` was used in the sample code when Tod set up the Cliniko app 🤷‍♂️ 

More than happy to change it back to `support@`?

Reply:
Ah I gotcha, thanks for letting me know. I changed that to support@